src/main/kotlin/org/opensearch/reportsscheduler/action/ReportInstanceActions.kt

```diff
@@ -85,8 +85,8 @@ internal object ReportInstanceActions {
             Metrics.REPORT_PERMISSION_USER_ERROR.counter.increment()
             throw OpenSearchStatusException("Permission denied for Report Definition ${request.reportDefinitionId}", RestStatus.FORBIDDEN)
         }
-        val beginTime: Instant = currentTime.minus(reportDefinitionDetails.reportDefinition.format.duration)
-        val endTime: Instant = currentTime
+        val beginTime: Instant = Instant.parse(reportDefinitionDetails.reportDefinition.format.timeFrom)
+        val endTime: Instant = Instant.parse(reportDefinitionDetails.reportDefinition.format.timeTo)
```

Comment:
for existing reports, are these fields present? If not, then this will break existing reports?